engine/src/main/java/org/openobservatory/engine/OONICheckInConfig.java

```diff
@@ -55,7 +55,7 @@ public OONICheckInConfig(String softwareName,
         this.softwareVersion = softwareVersion;
         webConnectivity = new CheckInConfigWebConnectivity();
         for (String category : categories) {
-            this.webConnectivity.add(category);
+            this.webConnectivity.addCategory(category);
```

Comment:
Change from `add` to `addCategory` https://github.com/ooni/probe-cli/blob/master/pkg/oonimkall/session.go#L346 not available in engine . Causing build failure